### ISSUE TITLE
Link submission mistake

### ISSUE THREAD

**Issue by ogun0054** [ogun0054/mtm6201-midterm#2]
@wendywarren

Thank you for your feedback on the mid-term assignment; I appreciate it. Regarding the submission link, I understood we should submit our GitHub repository page. Unfortunately, I mistakenly submitted the repository for the final as well. This may affect the grading of the final too, and I may not achieve the A I was hoping for. Can I resubmit it before you grade it? I would be very grateful for your support.
This is it here : https://ogun0054.github.io/mtm6201-final/

**Comment by wendywarren** [comment]
Absolutely! I will be grading your final tomorrow. Just update Brightspace.
